apps/src/lib/ui/lti/link/LtiLinkAccountPage/cards/NewAccountCard/index.tsx

```diff
@@ -42,6 +44,22 @@ const NewAccountCard = () => {
     }
   };
 
+  const handleNewAccountSubmit = async () => {
+    setIsSaving(true);
+
+    fetch('/lti/v1/account_linking/new_account', {
```

Comment:
Since we're in an async function, is it cleaner to await the fetch? Also, do we need to add handling for a non-200 response? Uncaught front-end errors won't notify via Honeybadger IIRC

Reply:
The await/async boils down the same stuff in the end and I think in this case it doesn't matter which we use. I went with Promise because I was going to chain, but ended up not needing to. If you feel strongly about await I can change it up, though it shouldn't matter much in this case.

Reply:
Nah purely a conventional preference; I'm fine rolling with this since we're on the clock